empress_gui.py

```diff
@@ -219,9 +219,9 @@ def init_windows(self):
         self.cost_space_window = None
         self.entire_space_window = None
         self.set_num_cluster_window = None
+        self.view_solution_space_window_after_setting_clusters = None
```

Comment:
Why did you rename `self.view_solution_space_window` to `self.view_solution_space_window_after_setting_clusters`? Is it much different from `self.solution_window`?

Reply:
Do you need a different solution window after setting clusters?

Reply:
self.view_solution_space_window_after_setting_clusters gives the solution space and self.solution_window gives the solutions for each cluster so they are different. I renamed self.view_solution_space_window to self.view_solution_space_window_after_setting_clusters because I got confused by the name without specifying that it is "after_setting_clusters" several times and it was kind of annoying.

Reply:
I think I can change self.solution_window to self.solution_window_one_per_cluster.

Reply:
Is there a way for the two variables to share the same code?

Also, I think it's generally good to have the variable name be more about what the variable is and less about when we need to use the variable. I'm still not sure what the code is doing, so I cannot suggest a new name.

Reply:
They cannot share the same code. You can see what they do in the two classes: class SolutionSpaceWindow (for self.view_solution_space_window_after_setting_clusters) and class ReconciliationsOnePerClusterWindow (for self.solution_window_one_per_cluster). SolutionSpaceWindow draws the solution space histogram and ReconciliationsOnePerClusterWindow draws the reconciliation graphs for clusters. 
This is an example of the solution space histogram:
<img width="903" alt="Screen Shot 2020-07-21 at 2 37 55 AM" src="https://user-images.githubusercontent.com/54883284/87973723-87abb280-cafb-11ea-9d1d-bc6a4ed22020.png">

Reply:
Do you think it would be good to name `self.view_solution_space_window_after_setting_clusters` to `self.solution_space_for_clusters_window` and `self.solution_window_one_per_cluster` to `self.view_reconciliations_window` or something of this sort?

Reply:
How about self.solution_space_for_clusters_window and self.view_reconciliations_for_clusters_window (since there is also "One MPR" under "Veiw reconciliations")? :))